src/components/atoms/Input/index.tsx

```diff
@@ -0,0 +1,54 @@
+
+import { cn } from "@/lib/utils";
+import * as React from "react"
+
+import { VariantProps, tv } from "tailwind-variants";
+
+const input = tv({
+  base: "flex w-full text-base rounded-md border border-input bg-transparent px-3 py-1 shadow-sm transition-colors file:border-0 file:bg-transparent file:text-sm file:font-medium placeholder:text-muted-foreground focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-gray-500 disabled:cursor-not-allowed disabled:opacity-50  invalid:ring-red-500 invalid:ring-1",
+  variants: {
+    variant: {
+      small: "h-9",
+      default: "h-11",
+      large: "h-14",
+    },
+  },
+  defaultVariants: {
+    variant: "default",
+  },
+})
+type InputVarinats = VariantProps<typeof input>;
```

Comment:
@Binabh everything else looked good while I tested but got a small typo in `InputVarinats`. Could you please fix this 🙇

Reply:
Thank you for noticing @itSubeDibesh I have fixed it.